src/components/Login/RegisterForm.js

```diff
@@ -121,14 +121,18 @@ const RegisterForm = ({ dispatch }) => {
     if (validateForm(false)) {
       await dispatch(
         registerUserAsync(
-          userName,
-          fName,
-          lName,
-          email,
-          "None",
-          new Date(),
-          password,
-          false
+          {
+            userName: userName,
+            firstName: fName,
+            lastName: lName,
+            email: email,
+            postalCode: "None",
+            dateRegistered: new Date(),
+            password: password,
+            isGoogleUser: false,
+          },
+          modal.openedFrom,
+          itemDetail.ownerId
```

Comment:
clarification for this line as well please

Reply:
same as above